machAeroTutorials/overset_tips.rst

```diff
@@ -0,0 +1,145 @@
+.. _overset_tips:
+
+########################
+Tips and Troubleshooting
+########################
+
+Steps to Create an Overset Mesh
+===============================
+
+Here is a list of commonly used steps to create an overset mesh with the usual workflow in the MDO Lab:
```

Comment:
I am not sure what's the best place to put these steps. Do you think it would be better to put them around or below [here](https://github.com/mdolab/MACH-Aero/pull/53/files#diff-40fb869b2ec9cbd00097c316947486173234d52ee2c878418cf8e9ad1735abeaR8)?

Reply:
I agree, it was out of place. How does it look now?